### PR TITLE
Use Ubuntu Wily, systemd and GraphicsMagick 1.3.21

### PR DESCRIPTION
**Pull request by kenoir**
Uses a new Ubuntu `Wily` AMI and consequently uses `systemd` and `GraphicsMagick 1.3.21`.

Removes `logrotate` config as we are now logging to `syslog` which is already rotated.

Depends:
 - https://github.com/guardian/grid-infra/pull/236

### CODE REVIEW

File: packer/media-service/provisioning.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "variables": {
-        "machine_images_ami": "ami-6857e51b"
+        "machine_images_ami": "ami-161b9765"
```
Comment (kenoir) [inline packer/media-service/provisioning.json]: Bumps us to `wily`

File: packer/media-service/provisioning.json [hunk 2]
```diff
@@ -20,11 +20,6 @@
 
     "provisioners": [
     {
-        "type": "file",
```
Comment (kenoir) [inline packer/media-service/provisioning.json]: Logs now go to syslog, so already taken care of by logrotate.

File: packer/media-service/media-service-ami.sh [hunk 3]
```diff
@@ -8,10 +8,6 @@ function new_section {
 
 set -e
 
-new_section "Moving config files into place"
-mv /tmp/media-service_logrotate /etc/logrotate.d/media-service
-sudo chown root:root /etc/logrotate.d/media-service
```
Comment (NickPapacostas) [inline packer/media-service/media-service-ami.sh]: if it goes to syslog now, are we using the syslog rotator or logrotate (per your comment below)?
I know very little about this (i.e. the blog posts i just read), so maybe we just talk about it IRL

File: packer/media-service/provisioning.json (hunk 1 above)
Comment (NickPapacostas) [inline packer/media-service/provisioning.json]: just confirming, this defines the new base ami to build the packer-generated local ami right? "**Am i right**?"

**Comment by NickPapacostas** [comment]
👍  lez give it a shot ^_^

**Comment by akash1810** [comment]
Any reason why this only affects cropper?

It feels like this change would put the packer provisioning.json in an odd, cropper specific, state?

We're also not explicitly installing a specific version of GM as part of the packer build... I guess 1.3.21 is latest?

**Comment by kenoir** [comment]
This change only affects cropper as making a blanket change to all apps
would be very disruptive. Others will follow. 1.3.21 is latest for Wily.
Specifying a version sounds like a good idea.
On Tue, 26 Apr 2016 at 22:55, Akash A <notifications@github.com> wrote:

> Any reason why this only affects cropper?
>
> It feels like this change would put the packer provisioning.json in an
> odd, cropper specific, state?
>
> We're also not explicitly installing a specific version of GM as part of
> the packer build... I guess 1.3.21 is latest?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1870#issuecomment-214899866>
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

File: packer/media-service/media-service-ami.sh (hunk 3 above)
Comment (kenoir) [inline packer/media-service/media-service-ami.sh]: logrotate already has config to rotate syslog logs!

File: packer/media-service/provisioning.json (hunk 1 above)
Comment (kenoir) [inline packer/media-service/provisioning.json]: Yup.

**Pull request merged**